src/Bicep.LangServer.IntegrationTests/GenerateParamsCommandTests.cs

```diff
@@ -49,9 +50,11 @@ public async Task GenerateParams_command_should_generate_paramsfile()
             await client.Workspace.ExecuteCommand(new Command
             {
                 Name = "generateParams",
-                Arguments = new JArray {
-                    bicepFilePath,
-                }
+                Arguments = JArray.FromObject(new object[] { new {
```

Comment:
[nit] Can we use `BicepGenerateParamsCommandParams` here rather than an untyped object? It should be possible to convert using something like `new JArray { params.ToJToken() }`

Reply:
makes total sense. updated untyped object usage to typed object usage 👍